### DISCUSSION TITLE
APK file where I set the KEYSTORE File when it is created

### DISCUSSION THREAD

**Discussion by bobwatcherx** [flet-dev/flet#2334] [Q&A]
### Question

like flutter when making it into an apk. There is a setting where it will generate a keystore file. which contains the keystore password file. but I didn't find that docs on the flet and can I retrieve the generated keystore results after the build is complete thx

### Code sample

```python
no code
```

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by FeodorFitsner** [comment]
https://github.com/flet-dev/flet/issues/2346
